src/app/components/PortraitVideoCarousel/PortraitVideoPromo/index.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render } from '#app/components/react-testing-library-with-providers';
+import Component from '.';
```

Comment:
Should we give this the same name as the file it's imported from?
```suggestion
import PortraitVideoPromo from '.';
```